integration-tests/utils/integration-helpers.js

```diff
@@ -105,6 +105,7 @@ const deleteWorkspace = withSignedInPage(async ({ page, billingProject, workspac
 
 const withWorkspace = test => async options => {
   console.log('withWorkspace ...')
+  console.log(options)
```

Comment:
is this change necessary/appropriate?

Reply:
Removed! Was an artifact of just inspecting the test!